src/app/collecting/collecting.component.ts

```diff
@@ -173,36 +173,36 @@ export class CollectingComponent implements OnInit, OnDestroy {
     this.p5.scale(-1, 1);
     this.p5.image(this.video, 0, 0, this.video.width, this.video.height);
 
-    if (this.pose) {
-      for (let i = 0; i < this.skeleton.length; i++) {
-        const a = this.skeleton[i][0];
-        const b = this.skeleton[i][1];
-        this.p5.strokeWeight(2);
-        this.p5.stroke(0);
-
-        this.p5.line(a.position.x, a.position.y, b.position.x, b.position.y);
-      }
-      for (let i = 0; i < this.pose.keypoints.length; i++) {
-        const x = this.pose.keypoints[i].position.x;
-        const y = this.pose.keypoints[i].position.y;
-        this.p5.fill(0);
-        this.p5.stroke(255);
-        this.p5.ellipse(x, y, 16, 16);
-      }
-    }
+    // if (this.pose) {
```

Comment:
Remove the comments if they're not relevant anymore.

Reply:
done